lib/kennel/models/project.rb

```diff
@@ -18,7 +18,10 @@ def self.file_location
       end
 
       def validated_parts
-        all = parts
+        # 'parts' called on same line as 'caller' for maximum cleanliness in strip_caller
+        here, all = [->() {caller}.call, parts]
+        all = [all] unless all.is_a?(Array)
```

Comment:
```suggestion
        here, all = [->() {caller}.call, parts.to_a]
```

Reply:
might be simpler to say `remove everything before __FILE__:__LINE__`

Reply:
The `all = [all]` was there for a purpose... however, this https://github.com/zendesk/kennel/pull/14227 (plus a follow-up grosser-kennel PR to fail if `parts` returns a non-array) is the better fix for that problem.